docs/framework/migration-guide/minimum-release-dword.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Minimum values of the Release DWORD for each .NET Framework version"
+ms.date: "04/02/2019"
+dev_langs: 
```

Comment:
you can remove dev_langs, You don't have any code here

Reply:
There is code (imported from samples, on lines 108-109). Does the dev_langs attribute apply only to inline code, or does it apply to external code as well, @mairaw?

Reply:
External too but this article has 30 lines and no code from what I can see

Reply:
Argh. Of course. I was looking at the code in the how to determine what versions are installed article.